shop/tests/test_views/test_feedback.py

```diff
@@ -1,12 +1,57 @@
+from enum import Enum, auto
+
 import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Count
 from django.urls import reverse
 from rest_framework import status
 
-from shop.models import Feedback
+from shop.models import Feedback, Image
```

Comment:
put imports in alphabet order

Reply:
"I" comes after "F", am I wrong?

Reply:
that is true, but in other places, you haven't followed this rule. for example in line below